Misc/NEWS.d/next/Core and Builtins/2022-05-02-20-19-06.gh-issue-86199.IZbF0m.rst

```diff
@@ -0,0 +1,6 @@
+`PyObject_Call` now guarantees any kwargs `dict` passed is shallow-copied
+before being passed to the callee, matching the documented equivalence with
+`callable(*args, **kwargs)`, fixing #86795.     When a function call's sole source of
+keyword arguments is a single `**`     unpacked `dict`, no copy is performed
+when the callable uses the vectorcall     protocol and no copy is needed to
+protect the caller's `dict` from mutation.
```

Comment:
I'm a little unclear on the proper docs formatting, but I'll see what I can do.

Reply:
See https://devguide.python.org/documenting/#additional-markup-constructs

Reply:
Yeah, I think this should be fixed now. It's been awhile since I used reST, forgot the rules. I cleaned it up a bit to match other changelog entries better as well.